Resources/Private/JavaScript/Host/Containers/LeftSideBar/PageTree/Node/index.js

```diff
@@ -86,16 +92,31 @@ export default class Node extends Component {
         onNodeToggle(item.contextPath);
     }
 
+    handleDoubleClick() {
+        console.log('onDoubleClick');
+    }
+
     handleNodeClick() {
         const {item, onNodeFocus} = this.props;
-
-        onNodeFocus(item.contextPath);
+        onNodeFocus(item.uri);
     }
 
     handleNodeLabelClick() {
-        const {item, onNodeClick, onNodeFocus} = this.props;
+        const {item, onNodeFocus, onNodeClick} = this.props;
 
         onNodeFocus(item.contextPath);
-        onNodeClick(item.uri);
+        onNodeClick(item.uri, item.contextPath);
+
+        if (this.props.item.isFocused) {
```

Comment:
Why do you put all of these `console.info` in here? If it's just for debugging, then let's not commit it.